src/System.Windows.Forms/src/System/Windows/Forms/OLE/DataObject.cs

```diff
@@ -127,6 +127,28 @@ public DataObject(object data)
 
     internal DataObject(string format, bool autoConvert, object data) : this() => SetData(format, autoConvert, data);
 
+    /// <summary>
+    ///  Flags that the original data was wrapped for clipboard purposes.
+    /// </summary>
+    internal bool IsWrappedForClipboard { get; set; }
```

Comment:
```suggestion
    internal bool IsWrappedForClipboard { get; init; }
```

I assume this should be an immutable value.

Reply:
Good catch!